src/op/compute_op.cc

```diff
@@ -424,12 +424,14 @@ ComputeLoopNest ComputeLoopNest::make(
   ret.main_nest = op::MakeLoopNest(
       stage, dom_map, 0, false, std::unordered_set<IterVar>(), &ret.main_vmap,
       debug_keep_trivial_loop);
+#if 0
```

Comment:
why?

Reply:
This is to remove the conservative condition check when the dimension is symbolic such as sequence in our case, which prevent the tensorization happen. 
As I checked last time, there is a gap from build_config to config at this stage, I probably will create another extended function with a parameter to control later. Currently I do not got time to dig through that. But currently this walkaround works.

Reply:
This change may have unexpected side effects. Please consider to scope it down to only the case you have problem with...

---
In reply to: [303141673](https://github.com/microsoft/onnxruntime-tvm/pull/2#discussion_r303141673) [](ancestors = 303141673)

Reply:
Like I mentioned, I do not know how long it could takes to scope it down, since it takes time to dig through and separate them. It could take days and may not work. But currently it works, so let's not block the release and create an work item for this.

Reply:
Update.